shared/src/simulation/events/transfer-specific-vehicle-request.ts

```diff
@@ -35,11 +38,13 @@ export class TransferSpecificVehicleRequestEvent implements SimulationEvent {
      */
     constructor(
         vehicleId: UUID,
+        transferInitiatingRegionId: UUID | undefined,
```

Comment:
```suggestion
        transferInitiatingRegionId?: UUID,
```

For consistency with the class member.